src/NexusMods.App.UI/Pages/LoadoutGrid/Columns/ModEnabled/ModEnabledViewModel.cs

```diff
@@ -24,57 +27,49 @@ public class ModEnabledViewModel : AViewModel<IModEnabledViewModel>, IModEnabled
     public ModStatus Status { get; set; } = ModStatus.Installed;
 
     [Reactive]
-    public ReactiveCommand<bool, Unit> ToggleEnabledCommand { get; set; }
+    public ICommand ToggleEnabledCommand { get; set; }
 
     [Reactive]
-    public ReactiveCommand<Unit, Unit> DeleteModCommand { get; set; }
```

Comment:
Should stay as reactive commands.

Reply:
I guess this was also something that was reverted with merge conflicts

Reply:
@erri120 how do we get that to work with .BindCommand? I couldn't get that to typecheck with ReactiveCommands because it complained that ReactiveCommand<> wasn't a ICommand so the call to `.BindCommand` failed.

Reply:
Used ReactiveCommand vs IReactiveCommand as Al suggested, and that worked

Reply:
In the VM interface and class you should use the concrete type `ReactiveCommand<TInput, TOutput>` (replace `TInput` and `TOuput` with what makes sense or `System.Reactive.Unit` if it doesn't have an input/output).